src/Tribe/Generator/Event.php

```diff
@@ -109,13 +114,158 @@ public function random_event_data( $fromDate, $toDate, $is_virtual ) {
                     '_tribe_events_zoom_join_url'               => 'https:\/\/zoom.us\/j\/1100000',
                     '_tribe_events_zoom_join_instructions'      => 'https:\/\/support.zoom.us\/hc\/en-us\/articles\/201362193-Joining-a-Meeting', 
                     '_tribe_events_zoom_global_dial_in_numbers' => ['+1 1000200200' => 'US', '+1 6699006833' => 'US']
-                ];
+                ] );
             }
+        }
+
+		if( $is_recurring ) {
+		    //Force recurring events creation in foreground
+            $lots_of_them = static function () {
```

Comment:
You never use this.
Also, you do realize that unfiltered this will create (theoretically) 2,147,483,647 instances? If it's supposed to do what I think - but you don't use it.

Reply:
This is to make recurring events be created in the foreground immediately upon request, and not be batched and sent into the background for processing later on.

Reply:
Also, this is copy pasta from @lucatume! XD